File: Walking/Assets/Scripts/DeathInfo.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class DeathInfo {
    public int SimulationId;
    public float DeathTime;
    public string BehaviourName;
    public string ActionName;
    public int Floor;
    public bool InRoom;

    public string SaveToString()
    {
        return JsonUtility.ToJson(this);
    }
}

File: statistics/deathinfo.py

class DeathInfo:
    def __init__(self, simulationId, deathTime, behaviourName, actionName, floor, inRoom):
        self.simulationId = simulationId
        self.deathTimeSeconds = deathTime
        self.deathTimeMinute = int(int(deathTime) / 60) + 1
        self.behaviourName = behaviourName
        self.actionName = actionName
        self.floor = floor
        self.inRoom = inRoom
